.eslintrc.json

```diff
@@ -73,6 +73,25 @@
     "formatjs/enforce-placeholders": ["off"],
     "formatjs/no-camel-case": ["off"],
     "formatjs/no-emoji": ["off"],
-    "formatjs/no-multiple-plurals": ["off"]
+    "formatjs/no-multiple-plurals": ["off"],
+
+    // styled-components-a11y
+    // TODO: needs to be removed after fixing the issues
+    "jsx-a11y/anchor-is-valid": 1,
```

Comment:
I find that using "warn" instead of `1` is clearer, especially because doesn't require any specific knowledge about ESLint to understand it. It's also the pattern we use above, so I think we should improve the consistency here.

Reply:
@Betree : Thanks. I've changed that to `warn` for consistence. I think now it looks better. 👍🏽